src/utils/numberStringHelpers.ts

```diff
@@ -450,11 +451,13 @@ export const formatNumberStringWithDigits = (
 /**
  * Ensures input function can handle negative numbers, by passing absolute values, and adding sign after execution.
  */
-const handleNegativeNumbers = (f: (v?: string) => string) => (value = '') => {
-    const negative = value.startsWith('-');
-    const abs = value.replace('-', '');
-    return negative ? `-${f(abs)}` : f(value);
-};
+const handleNegativeNumbers =
+    (f: (v?: string) => string) =>
+    (value = '') => {
+        const negative = value.startsWith('-');
+        const abs = value.replace('-', '');
+        return negative ? `-${f(abs)}` : f(value);
+    };
```

Comment:
Is all this reformatting due to a newer eslint or what is going on?

Reply:
I guess 🤷